sdk/python/v1beta1/kubeflow/katib/api/katib_client_test.py

```diff
@@ -57,6 +57,27 @@ def get_observation_log_response(*args, **kwargs):
         )
 
 
+def create_namespaced_persistent_volume_claim_response(*args, **kwargs):
+    if kwargs.get("namespace") == "pvc creation failed":
+        raise Exception("PVC creation failed")
+    else:
+        return {"metadata": {"name": "tune_test"}}
+
+
+def list_namespaced_persistent_volume_claim_response(*args, **kwargs):
+    if args[0] == "pvc creation failed":
```

Comment:
Should you check namespace like you did in the `create_namespaced_persistent_volume_claim_response` func ?

Reply:
I initially tried to check the namespace in the same way, but it caused an error. The issue was that `create_namespaced_persistent_volume_claim_response` was called with keyword arguments:
```[python]
self.core_api.create_namespaced_persistent_volume_claim(
                    namespace=namespace,
                    body=training_utils.get_pvc_spec(
                        pvc_name=name,
                        namespace=namespace,
                        storage_config=storage_config,
                    ),
                )
```
However, `list_namespaced_persistent_volume_claim_response` was called with a positional argument:
```[python]
pvc_list = self.core_api.list_namespaced_persistent_volume_claim(
                    namespace
                )
```
I’ve now adjusted `list_namespaced_persistent_volume_claim_response` to use keyword arguments as well, for consistent namespace checking.